### ISSUE TITLE
Force the origin to start at 0?

### ISSUE THREAD

**Issue by SHAESEN2** [openpharma/visR#295]
Currently ggplot has an offset to start their labels. Is this considered good practice? Personally I would not like to see such plots when quantiles are added as it is difficult to see at which times these quantiles are reached. I propose to start at the true origin using + scale_x_continuous(expand = c(0, 0)) + scale_y_continuous(expand = c(0, 0)) in visR
@cschaerfe @bailliem @timtreis 

![image](https://user-images.githubusercontent.com/47894155/147454474-ea7dd9a1-b560-48e6-88ba-839782d9e413.png)

**Comment by bailliem** [comment]
I agree. I think this is related to the general defaults. See in the discussion here:
 https://statmodeling.stat.columbia.edu/2020/05/07/make-andrew-happy-ggplot-trick-axis-at-zero/

Happy to override the ggplot defaults.

**Comment by timtreis** [comment]
Personally very much in favour of this, but I feel like this borders into the previous discussion we had about changing defaults in plots, no? Back then we decided against applying a better default ggplot theme as to not surprise the user with styling that was not actively asked for. If we take the step to adjust the lower axis limits, I think we can sneak in a better theme there as well.

**Comment by SHAESEN2** [comment]
Let us go with the expand option into visr.survfit and apply theme_bw as it originally was if everyone agrees.
